community/modules/remote-desktop/chrome-remote-desktop/main.tf

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+locals {
+
+  user_startup_script_runners = var.startup_script == null ? [] : [
+    {
+      type        = "shell"
+      content     = var.startup_script
+      destination = "user_startup_script.sh"
+    }
+  ]
+
+  configure_nvidia_driver_runners = var.install_nvidia_driver == false ? [] : [
+    {
+      type        = "ansible-local"
+      content     = file("${path.module}/scripts/configure-grid-drivers.yml")
+      destination = "/usr/local/ghpc/configure-grid-drivers.yml"
+    }
+  ]
+
+  configure_chrome_remote_desktop_runners = var.configure_chrome_remote_desktop == false ? [] : [
+    {
+      type        = "ansible-local"
+      content     = file("${path.module}/scripts/configure-chrome-desktop.yml")
+      destination = "/usr/local/ghpc/configure-chrome-desktop.yml"
+    }
+  ]
+
+  metadata = merge(var.metadata)
```

Comment:
this can be removed altogether and line 74 can be changed to `metadata = var.metadata`.

Reply:
done